presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -172,6 +173,12 @@ private static void setupLogging()
         Logging logging = Logging.initialize();
         logging.setLevel("org.apache.parquet.hadoop", WARN);
         logging.setLevel("parquet.hadoop", WARN);
+        logging.setLevel("org.glassfish.jersey.internal.inject.Providers", ERROR);
```

Comment:
perhaps, arrange these lines in alphabetical order:

```
        logging.setLevel("com.facebook.presto.security.AccessControlManager", WARN);
        logging.setLevel("com.facebook.presto.server.PluginManager", WARN);
        logging.setLevel("io.airlift.bootstrap.LifeCycleManager", WARN);
        logging.setLevel("org.apache.parquet.hadoop", WARN);
        logging.setLevel("org.eclipse.jetty.server.handler.ContextHandler", WARN);
        logging.setLevel("org.eclipse.jetty.server.AbstractConnector", WARN);
        logging.setLevel("org.glassfish.jersey.internal.inject.Providers", ERROR);
        logging.setLevel("parquet.hadoop", WARN);         logging.setLevel("parquet.hadoop", WARN);
```

Reply:
@mbasmanova Masha, do you think we should pass a parameter to createQueryRunner() from the tests that creates a DistributedQueryRunner, so that if a user starts HiveQueryRunner.main() on his/her laptop, he/she still see the full logging?